taskcluster/rb_taskgraph/job.py

```diff
@@ -96,7 +95,14 @@ def _extract_gradlew_command(run):
         _generate_secret_command(secret) for secret in run.get("secrets", [])
     ]
 
-    gradle_command = ["./gradlew"] + run.pop("gradlew")
+    maven_dependencies_dir = path.join(fetches_dir, "android-gradle-dependencies")
+    gradle_repos_args = [
+        "-P{repo_name}Repo=file://{dir}/{repo_name}".format(
```

Comment:
I checked with @pocmo and we agreed this was the most efficient way to pass variables to gradle.